doc/en/user/source/services/wms/webadmin.rst

```diff
@@ -179,3 +179,14 @@ Remote styles max request time and connection timeout can be configured in milli
 **Timeout in milliseconds** -The max connection timeout in milliseconds for remote style requests.
 
 **Max request time in milliseconds** -The max request time limit in milliseconds for remote style requests.
+
+Mark Factory Optimization
+----------------------------------------------------
+
+Mark factories can be filtered and ordered during the rendering execution. This makes room to optimize the rendering phase by omitting unused mark factories and prioritizing the fastest ones.
+
+.. figure:: img/service_WMS_markfactory.png
```

Comment:
caption:
```rst
.. figure:: img/service_WMS_markfactory.png
   
  WMS  MarkFactory Optimization
```

Reply:
I would of indicated "Mark Factory Precedence" or "Mark Factory Order", as optimization is the result of choosing a good order.

Reply:
Makes sense, change added.